gc/base/MarkMap.hpp

```diff
@@ -102,6 +103,19 @@ class MM_MarkMap : public MM_HeapMap
 		return _heapMapBaseDelta + (slotIndex << _heapMapIndexShift);
 	}
 
+	/*
+	 * check MarkMap if there is any liveObjects in the Card
+	 * this function assumes that card covers exactly 512 bytes.
+	 * @param heapAddress has to be card size aligned
+	 */
+	MMINLINE bool
+	areAnyLiveObjectsInCard(void* heapAddress)
```

Comment:
This function can be made const too:
```suggestion
	areAnyLiveObjectsInCard(void* heapAddress) const
```